extras/rootless/containerd-rootless-setuptool.sh

```diff
@@ -29,19 +29,15 @@ set -eu
 
 # utility functions
 INFO() {
-	# https://github.com/koalaman/shellcheck/issues/1593
-	# shellcheck disable=SC2039
-	/bin/echo -e "\e[104m\e[97m[INFO]\e[49m\e[39m ${*}"
+	printf "\e[104m\e[97m[INFO]\e[49m\e[39m %s\n" "$@"
```

Comment:
That is changing behavior.
We should stick with "$*" to keep the way this is working.

Reply:
I am not sure how "$@" is different from "${*}".

Reply:
`"$@"` will expand to `"$1" "$2"` etc
`"$*"` will expand to `"$1 $2"`

and `${*}` is just `$*`

For eg, try:


```
for i in "$@"; do
        echo "> $i"
done

for i in "$*"; do
        echo "> $i"
done

for i in "${*}"; do
        echo "> $i"
done
```

and call the above script with `./script.sh 1 2 3`